RNTester/android/app/BUCK

```diff
@@ -25,6 +25,7 @@ rn_android_library(
         react_native_target("java/com/facebook/react:react"),
         react_native_target("java/com/facebook/react/common:build_config"),
         react_native_target("java/com/facebook/react/modules/core:core"),
+        react_native_target("java/com/facebook/react/views/text:text"),
```

Comment:
Why was this dependency introduced?

Reply:
I reverted https://github.com/facebook/react-native/commit/983ba630254e2ec4b8013b8ff269115e2d73ff27 to start this pr so it came from that. I’m not sure why it was it was added in the original commit. Should I just remove it?